charts/mimir-distributed/values.yaml

```diff
@@ -716,6 +716,36 @@ query_frontend:
 store_gateway:
   replicas: 1
 
+  rollout_operator:
+    enabled: false
+    max_unavailable: 10
+    zones:
+    - name: zone-a
+      nodeSelector:
+        topology.kubernetes.io/zone: zone-a
```

Comment:
Let's have this commented out, and have affinity defined for all three zones. It's going to be the same independent from the node selector